spec/components/work_packages/date_picker/dialog_content_component_spec.rb

```diff
@@ -52,6 +52,12 @@
       it "shows the date form" do
         expect(dialog_content).to have_field(I18n.t("attributes.start_date"))
         expect(dialog_content).to have_field(I18n.t("attributes.due_date"))
+        expect(dialog_content).to have_css(
+          'input[data-test-selector="op-datepicker-modal--start-date-field"]' \
+          '[aria-live="polite"]'
+        )
+        expect(dialog_content).to have_css('input[data-test-selector="op-datepicker-modal--due-date-field"][aria-live="polite"]')
+        expect(dialog_content).to have_css('input[data-test-selector="op-datepicker-modal--duration-field"][aria-live="polite"]')
```

Comment:
Even better (more DRY) would be the following:

```suggestion
        expect(dialog_content).to have_field(I18n.t("attributes.start_date"), aria: { live: :polite })
        expect(dialog_content).to have_field(I18n.t("attributes.due_date"), aria: { live: :polite })
        expect(dialog_content).to have_field(WorkPackage.human_attribute_name("duration"), aria: { live: :polite })
```